src/main/webapp/app/shared/model/cancer-type.model.ts

```diff
@@ -3,18 +3,16 @@ import { IAssociation } from 'app/shared/model/association.model';
 import { TumorForm } from 'app/shared/model/enumerations/tumor-form.model';
 
 export interface ICancerType {
-  id?: number;
-  code?: string | null;
-  color?: string | null;
-  level?: number;
-  mainType?: string;
-  subtype?: string | null;
-  tissue?: string | null;
-  tumorForm?: TumorForm;
-  children?: ICancerType[] | null;
-  synonyms?: ISynonym[] | null;
-  parent?: ICancerType | null;
-  associations?: IAssociation[] | null;
+  id: number;
+  code: string | null;
+  color: string | null;
+  level: number;
+  mainType: string;
+  subtype: string | null;
+  tissue: string | null;
+  tumorForm: TumorForm | undefined;
```

Comment:
This technically should not be undefined. Do you see otherwise?

Reply:
I wasn't sure what the default should be for `tumorForm` so I made [this change](https://github.com/oncokb/oncokb-transcript/pull/376/files#diff-46872c4ed9cdde649ee066ce4b1691794cf57530a281b543cefa0ae3da155385R639-R652) in [firebase-review-utils.tsx](https://github.com/oncokb/oncokb-transcript/blob/539c17494823f1ed786a5e72d260fb94ceb1c263/src/main/webapp/app/shared/util/firebase/firebase-review-utils.tsx#L639-L649) where I set it as undefined.

Reply:
You can use TumorForm.mixed

Reply:
done